blog-service/2024-04-26-apps.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Classic Apps to Next-Gen Apps Migration (Apps)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - apps
+  - app catalog
+  - classic-apps
+  - nex-gen-apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
```

Comment:
@jpipkin1 The feature was released on the date mentioned in the release note. Do we need to adjust the date to the date of the feature release or to the latest date of the release of release note?

Reply:
this is actually a continuous activity so we put today's date also as release note. We just want to let customers know that we have reached half mark so that they can install these V2 apps and improve their upgrade / versioning experience.

Reply:
@JV0812, as a rule, the date of the release note should be the date the release note is published, even if the feature released earlier. Customers monitor release notes on a daily basis for new announcements. If a release note suddenly appears dated days or weeks ago, they don't see it.